### DISCUSSION TITLE
add program opera gx and opera

### DISCUSSION THREAD

**Discussion by ImAsra** [ChrisTitusTech/winutil#2412] [Ideas]
Hi first of absolutely love your tool I've been looking for a ninite alternative because they never add new programs. only program i miss from your tool is opera gx and maybe opera for the none gamers. I'm sure there are others that would love to have it on there as well.

**Comment by og-mrk** [comment]
Hello @ImAsra 
Thanks for the suggestion, but unfortunately, we can't add OperaGX (or any Opera browser/product) to WinUtil, if you search up "Opera" in the `Pull Requests` tab, and look into the Closed ones.. you'll see the same/similar response from the Project Owner (Chris Titus), and from his response that he will never add any Opera Product to WinUtil, ever.

I hope this answers your question, and currently, there isn't an easy way of adding apps to WinUtil other than cloning the repo and modifying some files (the process _should_ already be documented, and it'll give you the ability if adding any app* that you want, and isn't found in WinUtil).

Of course, an idea in issue https://github.com/ChrisTitusTech/winutil/issues/2394 ( made by @MyDrift-user ) would be nice to have, so you can add your own list of apps (or even use other people's lists).. but it isn't implemented yet, nor can we guarantee that it'll be.

**Reply by ImAsra** [reply]
alright dont know wat kind of experience hes had with opera but ill respect his choice thanks for the info on adding it myself
